subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/derived/DerivedVariantsResolutionIntegrationTest.groovy

```diff
@@ -359,6 +359,7 @@ class DerivedVariantsResolutionIntegrationTest extends AbstractHttpDependencyRes
     // endregion
 
     // region Without Gradle Module Metadata
+    @IgnoreIf({ System.getProperty('org.gradle.integtest.force.realize.metadata') == 'true' })
```

Comment:
Nit: how about making it a constant?

Reply:
I tried extracting this check to a closure but annotations are finicky about compile-time constants.

How about the alternative version I just force-pushed:

```
import static org.gradle.api.internal.artifacts.dsl.DefaultComponentMetadataProcessor.FORCE_REALIZE
...
@IgnoreIf({ FORCE_REALIZE })
```

It might warn about accessing the private member `FORCE_REALIZE` but it's very clean otherwise!